src/dsp_tools/commands/xmlupload/models/ingest.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional
```

Comment:
Stylistic comment I would replace this with `| None = None`

Reply:
Thanks for the hint, fixed in acc0ed487fbdf176f2a62f0811d99cb212108b9a